Trappist/src/Promact.Trappist.Core/Controllers/TestConductController.cs

```diff
@@ -97,5 +254,18 @@ public TestConductController(ITestConductRepository testConductRepository, ITest
         }
         #endregion
         #endregion
+
+        #region Private Method
+        private async Task<bool> isAttendeeValid(int attendeeId)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename method 'isAttendeeValid' to match camel case naming rules, consider using 'IsAttendeeValid'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS100)